src/packages/galaxy/galaxy.py

```diff
@@ -60,6 +61,15 @@ class Galaxy:
     }
     MAX_PLOT_DISTANCE = 20000
 
+    MAX_RETRIES = 3
+    "The maximum number of times to retry a failed HTTP request before failing permanently."
+
+    TIMEOUT = ClientTimeout(total=10)
+    "A ClientTimeout object representing the total time an HTTP request can take before failing."
+
+    RETRY_SLEEP = 1
```

Comment:
[suggestion] implement exponential back off? 1 second between re-attempts may not be long enough...

Reply:
b4713e1